### PR TITLE
Add my new widgets project

### PR DESCRIPTION
**Pull request by rohitksp**

### CODE REVIEW

File: rohit/widgets/.eslintcache [hunk 1]
```diff
@@ -0,0 +1 @@
...se React?',\n    content: 'You use React by creating components'\n  }\n]\n\nexport default () => {\n  return (\n    <div>\n      <Search />\n   </div>\n  )\n}","/home/rohit/Documents/React-Learn/widgets/src/componets/Accordion.js",[],["25","26"],"/home/rohit/Documents/React-Learn/widgets/src/componets/Search.js",["27"],{"ruleId":"28","replacedBy":"29"},{"ruleId":"30","replacedBy":"31"},{"ruleId":"32","severity":1,"message":"33","line":5,"column":7,"nodeType":"34","messageId":"35","endLine":5,"endColumn":12},{"ruleId":"36","severity":1,"message":"37","line":20,"column":1,"nodeType":"38","endLine":26,"endColumn":2},{"ruleId":"28","replacedBy":"39"},{"ruleId":"30","replacedBy":"40"},{"ruleId":"32","severity":1,"message":"41","line":23,"column":11,"nodeType":"34","messageId":"35","endLine":23,"endColumn":20},"no-native-reassign",["42"],"no-negated-in-lhs",["43"],"no-unused-vars","'items' is assigned a value but never used.","Identifier","unusedVar","import/no-anonymous-default-export","Assign arrow function to a variable before exporting as module default","ExportDefaultDeclaration",["42"],["43"],"'timeoutId' is assigned a value but never used.","no-global-assign","no-unsafe-negation"]
```
Comment (nrchandan) [inline rohit/widgets/.eslintcache]: I guess we shouldn't commit this file.

File: rohit/widgets/src/componets/Accordion.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
```
Comment (nrchandan) [inline rohit/widgets/src/componets/Accordion.js]: There's a spelling mistake in the name of the "components" folder.

**Review by nrchandan — CHANGES_REQUESTED** [review]

File: rohit/widgets/.eslintcache (hunk 1 above)
Comment (rohitksp) [reply rohit/widgets/.eslintcache]: Okay

**Review by komal7292 — APPROVED** [review]

File: rohit/widgets/src/App.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+//import Accordion from './componets/Accordion';
+import Search from './componets/Search'
```
Comment (nrchandan) [inline rohit/widgets/src/App.js]: Folder name is still incorrect in the code.

File: rohit/widgets/src/components/Accordion.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+const Accordion = ({ items }) => {
+  
+  const [activeIndex, setActiveIndex] = useState(null)
+  
+  const onTitleClicked = (index) => {
+    setActiveIndex(index)
+  }
+
+  const renderedItems = items.map( (item, index)=> {
```
Comment (nrchandan) [inline rohit/widgets/src/components/Accordion.js:11]: Let's pay more attention to code formatting. There should be spaces around operators/symbols.

File: rohit/widgets/src/App.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+//import Accordion from './componets/Accordion';
```
Comment (nrchandan) [inline rohit/widgets/src/App.js:2]: Rohit, please pay attention when fixing errors. Fix not just one error or one line, but look around for more of such errors.

**Review by nrchandan — APPROVED** [review]

**Pull request merged**